prettier.config.mjs

```diff
@@ -2,8 +2,6 @@
  * @type {import('prettier').Config}
  */
 export default {
-  printWidth: 120,
-  singleQuote: true,
   overrides: [
```

Comment:
Zelfde hier, sorted is wat voor te zeggen, er is ook wat voor te zeggen om overrides pas na de normale settings te melden.

Reply:
hoeft niet van mij, dan is het hek van de dam :P